src/components/CSettingsItem.vue

```diff
@@ -26,18 +26,37 @@
         </v-row>
       </div>
     </div>
+
+    <template v-if="type !== 'default'" #append>
+      <v-switch v-model="managedValue" color="primary" hide-details></v-switch>
+    </template>
   </v-list-item>
 </template>
 
 <script setup lang="ts">
 import { mdiChevronRight } from "@mdi/js";
 import { RouteLocationRaw } from "vue-router";
 
+/**
+ * The managed value is a value which will be managed by the settings item itself.
+ *
+ * It will be changed accordingly for some types like the `switch`.
+ */
+const managedValue = defineModel<boolean>();
```

Comment:
switchValue or rename to type to toggle and name it toggleValue?

Reply:
But this will not only be the value for the switch. If we add another type, this should be used again

Reply:
If I remember correctly you plan to merge value and managedValue at some point anyways? Please create an issue for that, then this is fine

Reply:
yeah, I will probably refactor this whole component to make it easier to extend more types. See #105